CharlieBackend.Api.UnitTes/ThemeServiceTests.cs

```diff
@@ -20,12 +20,19 @@ public class ThemeServiceTests : TestBase
     {
         private readonly Mock<IThemeRepository> _themeRepositoryMock;
         private readonly IMapper _mapper;
-        private readonly Mock<INotificationService> _notificationServiceMock;
+        private ThemeService _themeService;
 
         public ThemeServiceTests()
         {
-            _notificationServiceMock = new Mock<INotificationService>();
             _mapper = GetMapper(new ModelMappingProfile());
+            _themeRepositoryMock = new Mock<IThemeRepository>();
+        }
+
+        private void Initialize()
+        {
+            _themeService = new ThemeService(
```

Comment:
You probably  can move it to constructor and not to invoke 'Initialize' every time